presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -0,0 +1,128 @@
+import androidx.compose.ui.unit.dp
+import me.androidbox.component.agenda.*
+import me.androidbox.component.general.PhotoPicker
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.backgroundBackColor
+import me.androidbox.component.ui.theme.backgroundWhiteColor
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun EventScreen(
+    eventScreenState: EventScreenState,
+    eventScreenEvent: (EventScreenEvent) -> Unit,
+    onEditTitleClicked: (title: String) -> Unit,
+    onEditDescriptionClicked: (description: String) -> Unit,
+    modifier: Modifier = Modifier) {
+
+    Scaffold(
+        modifier = modifier,
+        topBar = {
+            AgendaDetailTopBar(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(color = MaterialTheme.colorScheme.backgroundBackColor)
+                    .padding(horizontal = 16.dp),
+                editModeType = EditModeType.EditMode(),
+                displayDate = "31 March 2023", /* TODO Get the current date from the zoneDateTime and format it */
+                onCloseClicked = {  },
+                onEditClicked = {  },
+                onSaveClicked = {  })
+
+        },
+        content = {
+            Column(modifier = Modifier
+                .padding(it)
+                .fillMaxWidth()
+                .background(Color.Black)) {
+
+                Column (modifier = modifier
+                    .fillMaxWidth()
+                    .background(
+                        color = Color.White,
+                        shape = RoundedCornerShape(topStart = 32.dp, topEnd = 32.dp)
+                    )
+                    .padding(horizontal = 16.dp)
+                ) {
+                    Spacer(modifier = modifier.height(30.dp))
+                    AgendaHeader(
+                        agendaHeaderItem = AgendaHeaderItem.EVENT,
+                        subTitle = "Meeting",
+                        description = "This is the description",
+                        isEditMode = true,
+                        onEditTitleClicked = { newTitle ->
+                            onEditTitleClicked(newTitle)
+                        },
+                        onEditDescriptionClicked = {
+                            onEditDescriptionClicked("description")
+                        }
+                    )
+
+                    PhotoPicker(
+                        listOfPhotoUri = eventScreenState.listOfPhotoUri,
+                        onPhotoUriSelected = { uri ->
+                            eventScreenEvent(EventScreenEvent.OnPhotoUriAdded(uri))
+                        }
+                    )
+
+                    Spacer(modifier = modifier.height(26.dp))
+                    AgendaDuration(
+                        startTime = "08:00",
+                        endTime = "08:30",
+                        startDate = "Jul 21 2022",
+                        endDate = "Jul 21 2022",
```

Comment:
This can be obtained from the state

Reply:
@MKiperszmid Will be updating these in the next PR, currently I have them as mock data

Reply:
Gotcha